modules/@apostrophecms/area/ui/apos/components/AposAreaEditor.vue

```diff
@@ -196,6 +196,7 @@ export default {
     apos.bus.$on('widget-hover', this.updateWidgetHovered);
     apos.bus.$on('widget-focus', this.updateWidgetFocused);
     this.bindEventListeners();
+    document.documentElement.style.setProperty('--a-widget-margin', apos.ui.widgetMargin);
```

Comment:
Not where this should go, but where is correct?

For context, this is where we set/override a developer preference for the margin value as a CSS variable. This only needs to be set once per logged-in Edit mode-enabled view.

Reply:
It should be in something that only runs once per page load, probably.

Reply:
@abea yes, only logged in

Reply:
So AposAreas.js would probably make more sense.

Reply:
moved this to `AposAreas.js`